pycbc/progress.py

```diff
@@ -0,0 +1,12 @@
+last_update = -1.0
```

Comment:
I'm not sure a module should be dedicated to this. If this functionality is really needed, I would place it it in pycbc_inspiral directly. Also, the function needs a bit more fleshed out documentation string. 

See function sin https://github.com/ligo-cbc/pycbc/blob/master/pycbc/waveform/utils.py for examples.

@duncan-brown If this is moved to pycbc_inspiral, it's not really a global anymore. At the moment, it doesn't seem generic enough to be placed in its own module, though.